raiden/transfer/architecture.py

```diff
@@ -175,19 +175,34 @@ def __ne__(self, other):
 
 class ContractSendEvent(Event):
```

Comment:
I checked all the events, they all are calling `super().__init__` and using `super().__eq__`